src/universal/components/AppHeader/AppHeader.js

```diff
@@ -1,44 +1,43 @@
 import React, { Component } from 'react';
 import config from './../../../config';
 import { Link } from 'react-router';
-import LogoutButton from './../LogoutButton/LogoutButton';
 
 class AppHeader extends Component {
   static propTypes = {
     isLoggedIn: React.PropTypes.bool
   };
 
-  constructor(props, context) {
-    super(props, context);
+  getAppHeaderClassName(styles, isDrawerVisble) {
+    let className = styles.AppHeader;
+    if (isDrawerVisble) {
+      className += ' ' + styles.AppHeader__isDrawerVisible;
+    }
+    return className;
   }
 
-  getHomeLink() {
+  get homeLink() {
     return this.props.isLoggedIn ? '/app' : '/';
   }
 
   render() {
     const { isLoggedIn } = this.props;
+    const isDrawerVisble = isLoggedIn;
```

Comment:
I see the styles as a resource, no verbs needed. It is misleading to have a verb there in my opinion, because there is no predicate to decide if that thing is visible or not. I see it only as a resource to dictate how to render the element.